app/exts.py

```diff
@@ -0,0 +1,6 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+
+db = SQLAlchemy()
+migrate = Migrate(db=db)
```

Comment:
I've not seen this exts.py layout before. It's kinda neat, did you see it recommended somewhere?

Reply:
Yeah I basically just copied it from a tutorial. I think it looks nice though.